src/components/button.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = props => {
+  const { name, color, wide } = props;
+  const classes = color === 'white' && wide ? 'white wide' : color;
+  return (
+    <div className={classes}>{name}</div>
+  );
+};
```

Comment:
[Optional]

We could destructure the props `{ name, color, wide }` directly from the parameter, e.g. `const Button = ({ name, color, wide }) => {`